.github/workflows/nvd.yml

```diff
@@ -0,0 +1,12 @@
+name: NVD Scan
```

Comment:
You should specify here (if not in the filename) that this is a schedule scan

Reply:
https://github.com/yetanalytics/lrsql/pull/197/commits/fea48c7d6e6af3eacdee27cad067f6ca83fc8d90